doc/user-guide/src/installation/other.md

```diff
@@ -29,6 +29,32 @@ $ curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --default
 $ curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --profile minimal --default-toolchain nightly
 ```
 
+
+## Using a package manager
+
+> Please note that the rustup project is not maintaining any package mentioned in this section.
+> If you have encountered any problems installing `rustup` with a package manager,
+> please contact the package maintainer(s) for further information.
```

Comment:
I added this section hoping to reduce responsibility confusions.

Reply:
How does this display in the rendered documentation?

Reply:
@djc If you mean the quote-as-callout part, have a look at the bottom of https://doc.rust-lang.org/stable/book. I got my inspiration from there.

Reply:
Okay, looks nice! I was wondering if mdBook natively has something like "warning" blocks which I think I've seen in similar context, but quick skimming of the mdBook docs didn't turn up anything like that.